src/idiomatic/leveraging-the-type-system/extension-traits/extending-other-traits.md

```diff
@@ -0,0 +1,86 @@
+---
+minutes: 15
+---
+
+# Extending Other Traits
+
+Extension traits can attach new methods to _all_ implementors of a given trait:
+
+```rust
+mod ext {
+    use std::fmt::Display;
+
+    pub trait DisplayExt {
+        fn quoted(&self) -> String;
+    }
+
+    impl<T: Display> DisplayExt for T {
+        fn quoted(&self) -> String {
+            format!("'{}'", self)
+        }
+    }
+}
+
+pub use ext::DisplayExt as _;
+
+assert_eq!("dad".quoted(), "'dad'");
+assert_eq!(4.quoted(), "'4'");
+assert_eq!(true.quoted(), "'true'");
```

Comment:
This isn't runnable, is it?

Reply:
It is, why not?

Reply:
Because the `assert_eq!` should be in a function.

Reply:
The Rust playground will implicitly wrap them in a `main` function!